bchrpc/proxy/gw.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+
+	gw "github.com/gcash/bchd/bchrpc/proxy/gen"
+)
+
+var (
+	// command-line options:
```

Comment:
NIT: These comments are self evident and don't provide a lot of value.

Reply:
done.